index.bs

```diff
@@ -1507,6 +1505,81 @@ these steps:
 
 </div>
 
+
+## Drag and Drop ## {#drag-and-drop}
+
+<xmp class=idl>
+partial interface DataTransferItem {
+    FileSystemHandle? getAsFileSystemHandle();
+};
+</xmp>
+
+During a <em>drag-and-drop operation</em>, dragged file and
+directory items are associated with [=file entries=] and [=directory entries=] 
+respectively.
+
+<div class="note dominto">
+: |handle| = item . {{getAsFileSystemEntry()}}
+:: Returns a {{FileSystemFileHandle}} object if the dragged item is file and a {{FileSystemDirectoryHandle}} object if the dragged item is a directory.
+</div>
+
+<div algorithm>
+
+The <dfn method for=DataTransferItem>getAsFileSystemEntry()</dfn> method steps are:
+
+1. If the {{DataTransferItem}} object is not in the <a spec=html>read/write
+    mode</a> or the <a spec=html>[=read-only mode=]</l>, then return null.
+
+1. If the <a spec=html>[=the drag data item kind=]</a> is not <em>File</em>, then return
```

Comment:
you don't need `<a>` and `[=`. Just using `<a>` is the same as using [= already.